content/container-environment.md

```diff
@@ -12,13 +12,15 @@ Galaxy runs your app in a set of containers using the [Docker](https://www.docke
 
 <h2 id="network">Network environment</h2>
 
-Galaxy runs your app containers in a firewalled network environment.  Only one port is exposed for external connections.  Galaxy tells your app what port to listen on via the `$PORT` environment variable, which contains a number such as `3000`.
+Galaxy runs your app containers in a firewalled network environment.  Only one port is exposed for external connections.  Galaxy tells your app what port to listen on via the `$PORT` environment variable, which contains a number such as `3000`. You can set this number by setting the `$PORT` environment variable.
 
-Galaxy forwards HTTP connections (port 80) on your app's configured [domains](/custom-domains.html) to this port, as well as HTTPS connections (port 443) if you've [configured encryption](/encryption.html).  You cannot serve connections on any other ports.
+Galaxy forwards HTTP connections (port 80) on your app's configured [domains](/custom-domains.html) to the port exposed for external connections. HTTPS connections (port 443) are also forwarded to this port if you've [configured encryption](/encryption.html).  You cannot serve connections on any other ports.
 
-When your app connects to other services like your database, its connections will always appear to come from one of a fixed set of IP addresses.  (These IP addresses are not the IPs of the individual machines your container runs on, so don't be surprised if they don't match.  These addresses are also distinct from the addresses that our "ingress" DNS address points to --- don't point your DNS at these IP addresses!)
+When your app connects to other services like your database, those service's connections will always appear to come from one of a fixed set of IP addresses.  (These IP addresses are not the IPs of the individual machines your container runs on, so don't be surprised if they don't match.  These addresses are also distinct from the addresses that our "ingress" DNS address points to --- don't point your DNS at these IP addresses!)
 
-Some services can be configured to only allow access from a list of IP addresses, so for an extra layer of security you can use these IPs in a "whitelist" on that service.  Note that these IP addresses are shared between all Galaxy customers, so you should still control access to your services by other means; whitelisting is just a defense-in-depth approach that can protect your app from non-targeted attacks.
+<h2 id="whitelisting">Whitelisting</h2>
```

Comment:
maybe "IP Whitelisting"?

Also, the previous paragraph is part of this section for sure.  And I do think this goes under "network environment" (the layout will make more sense when there are other kinds of environment described...)